test/Runtime/loaded_image_uniqueness.swift

```diff
@@ -34,7 +39,11 @@ do {
     guard let section = _getMetadataSection(i) else {
       fatalError("Section \(i) failed to resolve.")
     }
-    let name = String(cString: _getMetadataSectionName(section))
+    let cName = _copyMetadataSectionName(rawPointer)
+    defer {
+      _freeMetadataSectionName(cName)
+    }
+    let name = cName.flatMap { String(validatingUTF8: $0) } ?? "<unavailable>"
```

Comment:
Nit:`<unknown>` might be better to match what we do for symbols which we cannot symbolize?